deltacat/io/read_api.py

```diff
@@ -3,7 +3,7 @@
 from deltacat.utils.common import ReadKwargsProvider
 
 from ray.data import read_datasource
-from ray.data.impl.arrow_block import ArrowRow
+from ray.data._internal.arrow_block import ArrowRow
```

Comment:
Ideally, we should not rely on imports from ray _internal modules as they are subject to change without notice. Is ArrowRow not part of public module?

Reply:
Agreed, but unfortunately `ArrowRow` isn't part of any ray public module and is only located in `_internal`

Reply:
Ok, maybe you want to add a TODO to clean up this once we have this type available in public module?